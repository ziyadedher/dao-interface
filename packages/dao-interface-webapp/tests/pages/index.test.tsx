/**
 * @jest-environment jsdom
 */

/* eslint-disable import/no-extraneous-dependencies -- ignore extraneous dependencies in test files. */

import { render } from "@testing-library/react";

import Index from "../../src/pages";

describe("index page", () => {
  test("renders", () => {
    const result = render(<Index />);
    expect(result).toMatchSnapshot();
  });
});
