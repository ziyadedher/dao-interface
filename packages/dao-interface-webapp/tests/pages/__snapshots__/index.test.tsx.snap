// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index page renders 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="w-full min-h-screen bg-gray-50"
      >
        <div
          class="flex flex-col justify-center items-center mx-auto"
        >
          <div
            class="flex flex-col gap-8 items-center p-8 w-full"
          >
            <h1
              class="text-4xl text-gray-800"
            >
              Proposals
            </h1>
            <div
              class="flex flex-row gap-8 w-full"
            >
              <div
                class="flex flex-col flex-1 w-1/2"
              />
              <div
                class="flex flex-col w-1/2"
              >
                <div
                  class="w-full bg-gray-50"
                >
                  <div
                    class="flex flex-col gap-4 justify-center items-center h-full text-gray-500"
                  >
                    <span>
                      <svg
                        fill="currentColor"
                        height="64"
                        viewBox="0 0 256 256"
                        width="64"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <rect
                          fill="none"
                          height="256"
                          width="256"
                        />
                        <path
                          d="M35.3,103.1a96.4,96.4,0,0,1,24.8-43"
                          fill="none"
                          stroke="currentColor"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-width="16"
                        />
                        <path
                          d="M60.1,195.9a95.7,95.7,0,0,1-24.8-43.1"
                          fill="none"
                          stroke="currentColor"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-width="16"
                        />
                        <path
                          d="M152.9,220.7a94.2,94.2,0,0,1-49.7,0"
                          fill="none"
                          stroke="currentColor"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-width="16"
                        />
                        <path
                          d="M220.7,152.9a96.4,96.4,0,0,1-24.8,43"
                          fill="none"
                          stroke="currentColor"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-width="16"
                        />
                        <path
                          d="M195.9,60.1a95.7,95.7,0,0,1,24.8,43.1"
                          fill="none"
                          stroke="currentColor"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-width="16"
                        />
                        <path
                          d="M103.1,35.3a94.2,94.2,0,0,1,49.7,0"
                          fill="none"
                          stroke="currentColor"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-width="16"
                        />
                      </svg>
                    </span>
                    <div
                      class="text-center"
                    >
                      <p
                        class="font-bold"
                      >
                        Select a proposal to get started.
                      </p>
                      <p>
                        Choose a proposal to browse from the selector.
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="w-full min-h-screen bg-gray-50"
    >
      <div
        class="flex flex-col justify-center items-center mx-auto"
      >
        <div
          class="flex flex-col gap-8 items-center p-8 w-full"
        >
          <h1
            class="text-4xl text-gray-800"
          >
            Proposals
          </h1>
          <div
            class="flex flex-row gap-8 w-full"
          >
            <div
              class="flex flex-col flex-1 w-1/2"
            />
            <div
              class="flex flex-col w-1/2"
            >
              <div
                class="w-full bg-gray-50"
              >
                <div
                  class="flex flex-col gap-4 justify-center items-center h-full text-gray-500"
                >
                  <span>
                    <svg
                      fill="currentColor"
                      height="64"
                      viewBox="0 0 256 256"
                      width="64"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <rect
                        fill="none"
                        height="256"
                        width="256"
                      />
                      <path
                        d="M35.3,103.1a96.4,96.4,0,0,1,24.8-43"
                        fill="none"
                        stroke="currentColor"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                        stroke-width="16"
                      />
                      <path
                        d="M60.1,195.9a95.7,95.7,0,0,1-24.8-43.1"
                        fill="none"
                        stroke="currentColor"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                        stroke-width="16"
                      />
                      <path
                        d="M152.9,220.7a94.2,94.2,0,0,1-49.7,0"
                        fill="none"
                        stroke="currentColor"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                        stroke-width="16"
                      />
                      <path
                        d="M220.7,152.9a96.4,96.4,0,0,1-24.8,43"
                        fill="none"
                        stroke="currentColor"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                        stroke-width="16"
                      />
                      <path
                        d="M195.9,60.1a95.7,95.7,0,0,1,24.8,43.1"
                        fill="none"
                        stroke="currentColor"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                        stroke-width="16"
                      />
                      <path
                        d="M103.1,35.3a94.2,94.2,0,0,1,49.7,0"
                        fill="none"
                        stroke="currentColor"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                        stroke-width="16"
                      />
                    </svg>
                  </span>
                  <div
                    class="text-center"
                  >
                    <p
                      class="font-bold"
                    >
                      Select a proposal to get started.
                    </p>
                    <p>
                      Choose a proposal to browse from the selector.
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
